### ISSUE TITLE
Tree Item - Use aria-hidden for SVGs - (2036570418)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5616]
### Module: 
 38 Tree and Tree Item 

### Violation: 
Hide SVGs from assistive technologies using the `aria-hidden` prop.

![image](https://user-images.githubusercontent.com/5023024/202574349-d447e3b7-b45f-4ea8-b16b-fe67ef56cef0.png)

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html) 

### Instance ID: 
 2036570418 

### Severity: 
 10 

### Description: 
 [Issue]
There are SVG images that serve a decorative purpose but are exposed to screen readers.

[User Impact]
Screen readers may announce <img> elements with no alt attribute by their filename and SVGs with no accessible name as "group." This gives screen reader users irrelevant and confusing information.

[Code Reference] 
 ```
<svg class="check-svg" viewbox="0 0 16 16"><path d=""></path></svg>
 ``` 

### Note: 
 [Suggestion]
Provide alternative text for images. Non-meaningful images must be marked as decorative, which causes screen readers to ignore them. To mark an ``` <svg> ``` element as decorative, add aria-hidden="true" to the element.

[Compliant Code Example] 
 ```
<svg aria-hidden="true" class="check-svg" viewbox="0 0 16 16"><path d=""></path></svg>
 ```
 

### Media Type: 
 Images 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Verified in `next.698`.
